docs/_docs/01_getting-started/quick-start.md

```diff
@@ -112,14 +112,18 @@ func TestWebServer(t *testing.T) {
     TerraformDir: "../web-server",
   }
 
-  // At the end of the test, run `terraform destroy`
-  defer terraform.Destroy(t, terraformOptions)
-
-  // Run `terraform init` and `terraform apply`
-  terraform.InitAndApply(t, terraformOptions)
+  terraform.UnitTest(t, &terraform.UnitTestConfig{
+    TerraformOptions: terraformOptions,

```

Comment:
I think this should be a function to generate the options, that takes in the test data.

Oftentimes the terraform options will depend on random data. E.g., you might use `random.UniqueId` to seed a name variable. When iterating locally, you want to ensure that you can repeatedly run apply using the same random values. In this workflow, I would want to:

- Use `setup` to define the random values and pack it into `TestData`
- In `init_apply`, load the test data and generate the options based on that

This way, whenever I want to iterate, I would skip the setup step after the first run so that I preserve all the randomly generated data across the different test runs.

Hope I'm making sense here!